### PR TITLE
let me inject my people

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
"""""""""""""Correct"""""""""""""""""""' aka rebalances medical injectors to let you inject people once more

### CODE REVIEW

File: code/modules/clothing/spacesuits/rig/modules/utility.dm [hunk 1]
```diff
@@ -263,20 +263,27 @@
 		list(/obj/item/reagent_containers/glass/beaker/large, "nutriment", 60),
 		list(/obj/item/reagent_containers/glass/beaker/large, "tricordrazine", 60)
 	)
+	interface_name = "integrated chemicalc combat dispenser"
```
Comment (ThePainkiller) [inline code/modules/clothing/spacesuits/rig/modules/utility.dm:266]:
```suggestion
	interface_name = "integrated chemical combat dispenser"
```

File: code/modules/clothing/spacesuits/rig/modules/utility.dm [hunk 2]
```diff
@@ -263,20 +263,27 @@
...
+	disruptive = 1
 	initial_beakers = list(
 		list(/obj/item/reagent_containers/glass/beaker/large, "bicaridine", 60),
 		list(/obj/item/reagent_containers/glass/beaker/large, "inaprovaline",60),
 		list(/obj/item/reagent_containers/glass/beaker/large, "kelotane",60),
 		list(/obj/item/reagent_containers/glass/beaker/large, "anti_toxin", 60),
 		list(/obj/item/reagent_containers/glass/beaker/large, "spaceacillin", 60)
 	)
+	interface_name = "integrated chemical injector"
+	interface_desc = "Dispenses loaded chemicals directly into the wearer's bloodstream or pacients."
```
Comment (ThePainkiller) [inline code/modules/clothing/spacesuits/rig/modules/utility.dm:286]:
```suggestion
	interface_desc = "Dispenses loaded chemicals directly into the bloodstream of its target. Can be used on the wearer as well."
```

File: code/modules/clothing/spacesuits/rig/modules/utility.dm [hunk 3]
```diff
@@ -263,20 +263,27 @@
 		list(/obj/item/reagent_containers/glass/beaker/large, "nutriment", 60),
 		list(/obj/item/reagent_containers/glass/beaker/large, "tricordrazine", 60)
 	)
+	interface_name = "integrated chemicalc combat dispenser"
+	interface_desc = "Dispenses loaded chemicals directly into the wearer's bloodstream."
```
Comment (ThePainkiller) [inline code/modules/clothing/spacesuits/rig/modules/utility.dm:267]:
```suggestion
	interface_desc = "Dispenses loaded chemicals directly into the user's bloodstream."
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]
Minor typo fixes

**Comment by Trilbyspaceclone** [comment]
Do to it being a bug fix no need to wait on it

**Pull request merged**